src/Compilers/VisualBasic/Test/Semantic/FlowAnalysis/FlowTestBase.vb

```diff
@@ -64,8 +64,14 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.UnitTests
             Return CompileAndGetModelAndSpan(program, Function(binding, startNodes, endNodes) AnalyzeDataFlow(binding, startNodes, endNodes), ilSource, errors)
         End Function
 
-        Protected Function CompileAndAnalyzeControlAndDataFlow(program As XElement, Optional ilSource As XCData = Nothing, Optional errors As XElement = Nothing) As Tuple(Of ControlFlowAnalysis, DataFlowAnalysis)
-            Return CompileAndGetModelAndSpan(program, Function(binding, startNodes, endNodes) Tuple.Create(AnalyzeControlFlow(binding, startNodes, endNodes), AnalyzeDataFlow(binding, startNodes, endNodes)), ilSource, errors)
+        Protected Function CompileAndAnalyzeControlAndDataFlow(
+                                                                program As XElement,
+                                                       Optional ilSource As XCData = Nothing,
+                                                       Optional errors As XElement = Nothing
+                                                              ) As (ControlFlowAnalysis As ControlFlowAnalysis, DataFlowAnalysis As DataFlowAnalysis)
```

Comment:
what is happening here?
1. please do not unnecessarily change formatting
2. please don't use formatting styles that are used *nowhere* else in roslyn.

Reply:
Using a tuple literals. Change styling back.